docs/reference/tab-widgets/quick-start-install.asciidoc

```diff
@@ -19,8 +19,9 @@ Desktop].
 +
 [source,sh,subs="attributes"]
 ----
+docker network create elastic
 docker pull {docker-repo}:{version}
-docker run --name es01-test -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" {docker-image}
+docker run --name es01-test --net elastic -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" {docker-image}
```

Comment:
Because the 8.0 image hasn't been released yet, this won't display in the preview. To test, you can use:

```
docker network create elastic
docker pull docker.elastic.co/elasticsearch/elasticsearch:7.12.0
docker run --name es01-test --net elastic -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:7.12.0
```